### DISCUSSION TITLE
Extension/torsion coupling stiffness

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#39] [VABS Related Discussions]
CAMRAD requires section property, KT:
•	extension/torsion coupling stiffness for the isotropic model (p. 202)
•	units: lb-ft (table on p. 203)
•	produces coupling between extension and bending (p. 203)

Where can the value for this property be found in VABS output?

Page numbers refer to CAMRAD II documentation, Volume VI, Rotorcraft Input.

**Comment by wenbinyu-PU** [comment]
I will check with Wayne Johnson to find out.
